static/defaults.config.php

```diff
@@ -59,6 +59,10 @@
 		// Only show small preview pictures.
 		'always_show_preview' => false,
 
+		// ap_always_bcc (Boolean)
+		// Adressing receivers via ActivityPub by BCC. Increases privacy, decreases performnce.
+		'ap_always_bcc' => false,
```

Comment:
Why not `activitypub.always_bc` instead of `config.ap_always_bc`?

Reply:
I thought about it - but normally the first value is always "system" for core stuff - and a different value mostly means that some addon is using this value.

Reply:
I'd like to revisit this policy, we can still have addons have their own namespace but have core config key names be sorted into larger features.

Reply:
Prior to the change of this policy, I would like to have a separated config functionality especially for addons. Just like now we do have "config" and "pconfig" we could have variations of this, especially for addons. So the config table could receive an additional parameter that indicates if this value is an addon value or a system value.

Reply:
Why? Since official addons have their own category, there will never be a conflict with core config keys as long as we review every single addon, which we do.

Reply:
I would like to have a clean solution.

Reply:
What do you mean by "clean"?

Reply:
For example we do have (P)Config values for the Diaspora addon with the category "diaspora", but we also could have some core config values for "diaspora". So either we need some additional parameter or naming scheme to mark a config value as a core value.

Reply:
No we don't, we control both the core repository and the diaspora addon repository. So we just have to make sure both config keys are different, 'tis all. It really is that simple since we aren't accepting user submissions sight unseen.

Reply:
Since this would be a significant change in how we are doing this, I would like to discuss it in the developer forum prior to apply any change. Then I would like to rework the whole naming scheme. This would mean a renaming of many existing config values (including some fallback mechanism for reading deprecated config values).

Reply:
Fair enough.